lib/core/src/lib/form/components/widgets/date-time/date-time.widget.ts

```diff
@@ -71,8 +70,16 @@ export class DateTimeWidgetComponent extends WidgetComponent implements OnInit,
         this.onFieldChanged(this.field);
     }
 
-    private patchFormControl(): void {
+    updateReactiveFormControl(): void {
+        this.updateFormControlState();
+        this.validateField();
+    }
+
+    private setFormControlValue(): void {
         this.datetimeInputControl.setValue(this.field.value, { emitEvent: false });
+    }
+
+    private updateFormControlState(): void {
```

Comment:
```suggestion
    private updateFormControlValidators(): void {
```
WDYT?
To me state is really broad term

Reply:
i think state in this case better describe what this method does, because it updates also readonly/disabled state (required is also state of the field)